src/ResourceManagement/Authorization/Authorization/Generated/AuthorizationManagementClient.cs

```diff
@@ -31,7 +31,8 @@ namespace Microsoft.Azure.Management.Authorization
     public partial class AuthorizationManagementClient : ServiceClient<AuthorizationManagementClient>, IAuthorizationManagementClient
     {
         private string _apiVersion;
-        
+        public const string APIVersion = "2015-07-01";
```

Comment:
It is edited, since lots of place use "2015-07-01", which is hard coded and not good for mainteance.